app_links/android/src/main/java/com/llfbandit/app_links/AppLinksPlugin.java

```diff
@@ -170,11 +171,9 @@ private boolean handleIntent(Intent intent) {
       return false;
     }
 
-    String action = intent.getAction();
-    String dataString = intent.getDataString();
+    AppLinksHelper helper = new AppLinksHelper();
```

Comment:
Methods on AppLinksHelper should be static:
- to improve performance
- to not ambiguate its responsability.